front/src/modules/rollingStock/helpers/utils.ts

```diff
@@ -198,22 +198,31 @@ export const orderSelectorList = (list: (string | null)[]) => {
     : list.sort();
 };
 
+/** This function will sort the curves list based on the number of conditions they have.  */
```

Comment:
You can create 1 unique comment, what do you think ?
```suggestion
/**
  * This function will sort the curves list based on the number of conditions they have.
  *
  * Curves will be sorted in the following order (comfort is compulsory): 
  * - curves with all conditions (power restriction & electrical profile)
  * - curves with at least power restriction
  * - curves with least electrical profile
  * - other curves (without power restriction and without electrical profile)
  */
```